### PR TITLE
Added recipe for cova package.

### PR DESCRIPTION
**Pull request by jkanem**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x ] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/cova/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+{% set version = "0.22.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cova-{{ version }}.1.tar.gz
+    sha256: 1e8f4dd144905b71eb228dda90a99d60d16bc7f6f40a997f70416347a362ee4f
+  - url: https://files.pythonhosted.org/packages/5b/78/0c8e4b41ee425be92888bc118af7c15fc7b771a7e495d7b48b223cd6c382/sentinel-0.3.0.tar.gz
```
Comment (xhochy) [inline recipes/cova/meta.yaml:11]: Please submit this as a separate recipe.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/cova/meta.yaml (hunk 1 above)
Comment (jkanem) [reply recipes/cova/meta.yaml:11]: Do you mean for the sentinel module (line 11)? I'd rather not become the maintainer of that package. We were planning on removing our dependence on it anyway. I'll change this recipe once we do. Can I leave this PR up, or do I need to delete it and make another one when the changes are ready? Thanks.

Comment (xhochy) [reply recipes/cova/meta.yaml:11]: I merged `sentinel` from someone else some hours ago, you should be able to directly depend on it.

Comment (jkanem) [reply recipes/cova/meta.yaml]: Awesome, thanks! I've taken it's installation out of the recipe and put it under requirements.

**Comment by jkanem** [comment]
@conda-forge-admin, please ping team

File: recipes/cova/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+about:
+  home: https://github.com/AgnostiqHQ/covalent
+  summary: Covalent Workflow Tool
+  license: AGPL-3.0-only
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - AgnostiqHQ
```
Comment (xhochy) [inline recipes/cova/meta.yaml:65]: This is not your username

**Review by carterbox — COMMENTED** [review]
Hi @jkanem, thanks for your patience.

Why did you decide to switch from `noarch: python` to linux only? covalent is advertised as python-only so it should be fine as noarch? The build logs from that old build are expired, but if it is because there are missing dependencies, our policy is that we publish as noarch anyways. The package will be uninstallable (so not broken) on win/osx or it will eventually be usable when builds for win/osx of thse dependencies make their way to conda-forge.

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Also, the covalent package on PYPI seems to be abandoned/unused. There is a process for claiming that package name if y'all want to go through the effort.
https://pypi.org/help/#project-name-claim

**Comment by jkanem** [comment]
@wjcunningham7, @venkatBala

**Comment by carterbox** [comment]
@jkanem, Do you want to add those additional recipes to this PR and update this cova recipe to the latest version?

**Comment by jkanem** [comment]
> @jkanem, Do you want to add those additional recipes to this PR and update this cova recipe to the latest version?

Yes, that's a route we might take. @wjcunningham7 and @venkatBala will be making that decision. That might not happen soon. But thanks for taking a look at the PR and reaching out to us.

**Pull request closed without merging**